src/deion/worker/util/defaultGameAttributes.js

```diff
@@ -103,6 +105,8 @@ if (process.env.SPORT === "football") {
         minRosterSize: 40,
         maxRosterSize: 53,
 
+        homeCourtAdvantage: 1,
```

Comment:
Doesn't need to be listed here if the value is the same in football and basketball.

Speaking of which... in the football GameSim/index.js file, can you make a similar change to apply this setting?

Reply:
Yup! Made change in commit https://github.com/dumbmatter/gm-games/pull/256/commits/43500fa07a58eab44290cbed6a2c92c46117350d